BackendServices/conf/routes

```diff
@@ -6,6 +6,8 @@
 GET     /                           controllers.Application.index()
 GET     /sayHello                   controllers.CompanyController.sayHello()
 GET		/company/info/:id			controllers.CompanyController.getInfo(id: Long)
```

Comment:
i see that you have create method in company, so it should be POST call to a different route.